frontend/src/scenes/insights/yAxisFormats.test.ts

```diff
@@ -0,0 +1,21 @@
+import { isYAxisFormat } from 'scenes/insights/yAxisFormat'
+
+describe('y axis formats', () => {
```

Comment:
Wooohoo tests. Could we maybe add some for the `formatYAxisValue` as well?

Reply:
> Not saying we have to but in the case of repeated styles, that could be pulled out to a variable rowClassNames to dry it up

The idea that seemed to be missing a name wasn't so much (IMHO) the list of classes but the idea that a config filter is wrapped in a particular way. So I extracted that... A bit more text but hopefully more helpful to the future traveller